pkg/sdk/testint/dynamic_table_integration_test.go

```diff
@@ -133,7 +134,7 @@ func TestInt_DynamicTableAlter(t *testing.T) {
 
 		err := client.DynamicTables.Alter(ctx, sdk.NewAlterDynamicTableRequest(dynamicTable.ID()).WithSuspend(sdk.Bool(true)).WithResume(sdk.Bool(true)))
 		require.Error(t, err)
-		expected := "alter statement needs exactly one action from: set, unset, refresh"
+		expected := fmt.Sprintf("exactly one of %s fileds %v must be set", "alterDynamicTableOptions", []string{"Suspend", "Resume", "Refresh", "Set"})
```

Comment:
why Sprintf and not ExactlyOneOf here? Because accessibility? Maybe just add a delegation to integration_test_imports.go for now, and we can think later, I would for sure prefer not to have repeated Sprintf like this here.

Reply:
Since this is now in testint, we cannot call errOneOf because it's in the SDK package. We can either export it or leave it as is

Reply:
kk declared in integration_test_imports.go and used in the test